core/templates/dev/head/pages/library-page/search-bar/search-bar.directive.html

```diff
@@ -64,7 +64,7 @@
     </ul>
   </div>
 </div>
-<div ng-class="{'open' : $ctrl.activeMenuName === 'language'}" uib-dropdown class="float-left oppia-navbar-button-container oppia-search-bar-language-selector protractor-test-search-bar-language-selector">
+<div ng-class="{'open' : $ctrl.activeMenuName === 'language', 'dropup' : $ctrl.dropup()}" uib-dropdown class="float-left oppia-navbar-button-container oppia-search-bar-language-selector protractor-test-search-bar-language-selector">
```

Comment:
@seanlip I use it here to enable the `dropup` class of `uib-dropdown`.

Reply:
Ah, got it. Thanks! I did not realize it was a Bootstrap thing.